src/components/Proposals/ProposalDetails/VoteCast.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { GU, IconCheck, RADIUS, textStyle, useTheme } from '@aragon/ui'
+const VOTE_YEA = 'VOTE_YEA'
+
+function VoteCast({ accountVote, balance, tokenSymbol }) {
+  const theme = useTheme()
+
+  return (
+    <div
+      css={`
+        display: flex;
+        justify-content: center;
+        margin-top: ${5 * GU}px;
+        background-color: ${theme.background};
+        border: 1px solid ${theme.border};
+        padding: ${4 * GU}px;
+        border-radius: ${RADIUS}px;
+      `}
+    >
+      <div
+        css={`
+          display: flex;
+          align-items: center;
+        `}
+      >
+        <div
+          css={`
+            display: flex;
+            align-items: center;
+            justify-content: center;
+            border-radius: 100%;
+            color: ${theme.accent};
+            border: 2px solid currentColor;
+            width: ${5.5 * GU}px;
+            height: ${5.5 * GU}px;
+            margin-right: ${2 * GU}px;
+            flex-shrink: 0;
+          `}
+        >
+          <IconCheck />
+        </div>
+
+        <div>
+          <h2
+            css={`
+              ${textStyle('body1')}
+              font-weight: 600;
+              margin-bottom: ${0.5 * GU}px;
+            `}
+          >
+            Your vote was cast successfully
+          </h2>
+          <p
+            css={`
+            ${textStyle('body2')}
+            color: ${theme.surfaceContentSecondary};
+          `}
+          >
+            You voted{' '}
+            <span
+              css={`
+                color: ${theme.surfaceContent};
+                font-weight: 600;
+                text-transform: uppercase;
+              `}
+            >
+              {accountVote === VOTE_YEA ? 'yes' : 'no'}
```

Comment:
As mentioned in a previous comment, we could pass this prop as a boolean and check 

```javascript
{accountHasVoted ? 'yes' : 'no'}
```

Then we could remove the `VOTE_YEA` string comparison

Reply:
here we are not checking if  the account has voted, but if the vote is  "yes" or "no". thats why voterInfo.hasVoted is not a boolean. It can be:  null,  'VOTE_YEA', 'VOTE_NO'